### PR TITLE
Update `parachain-template`

### PR DESCRIPTION
**Pull request by NachoPal**
This PR configures `parachain-template` to be ready to properly interact with the Common Goods parachains (only Statemint for now but more are coming)

It is unclear to me what configuration `parachain-template` was following.

This makes me think about the role of `rococo-parachain` runtime, whose configuration seems much closer to what we want to achieve. To discuss about it I created this issue with my view: https://github.com/paritytech/cumulus/issues/1222

I'll switch the PR to Draft while @gilescope finalise the XCM config

### CODE REVIEW

**Comment by NukeManDan** [comment]
This template is used to start most parachain projects from, and thus is intended to be _minimal_ AKAIU, thus adding more functionality here, especially divergent from the [substrate template](https://github.com/paritytech/substrate/tree/master/bin/node-template), isn't ideal.

If common good chains need a more purpose built template, I would think it should live independent from the soverign template (we have now) if significantly different. _Especially_ in the XCM configuration, because this template should never be set such that a soverign chain basing off the settings here unchanged could allow for the same trust with the relay and other siblings. Issues happened in  this light, addressed in https://github.com/paritytech/cumulus/pull/696 (at least this was attempted, not merged now I see :grimacing: )

**Comment by bkchr** [comment]
I'm here with @NukeManDan. These templates are about the bare minimum you need to run a parachain and that clearly doesn't involve communicating with any common good parachain. Which very probably also doesn't exist on local networks when you begin to code.

The template should not be misused to act as some kind of tutorial. Here you actually want to have some proper tutorial that could live on substrate docs. This would also be much better for explaining on what is what doing.

**Comment by NachoPal** [comment]
Ok, I understand, it can make sense to have a bare minimum parachain template. `parachain-template` will stay as it is.

Then I'll propose to make these changes to `rococo-parachain`. I see that actually it has already some XCM config to support Statemine.

Is it ok?

**Comment by bkchr** [comment]
> Ok, I understand, it can make sense to have a bare minimum parachain template. `parachain-template` will not be modified.
> 
> Then I'll propose to make these changes in `rococo-parachain`. I see that actually it has already some XCM config to support Statemine.
> 
> Is it ok?

If it is about testing. Makes sense.

File: parachain-template/node/src/command.rs [hunk 1]
```diff
@@ -23,14 +23,44 @@ use sp_runtime::traits::Block as BlockT;
 use std::{io::Write, net::SocketAddr};
 
 fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+	let (norm_id, para_id) = extract_parachain_id(id);
+
 	Ok(match id {
```
Comment (gilescope) [inline parachain-template/node/src/command.rs:28]: `match id`  should be `match norm_id`

**Pull request closed without merging**